app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,12 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+.CardList-card.is-closed
```

Comment:
Your `.CardList-card` is changing the appearance of the `.Discussion`, when it shouldn't. I think it's only supposed to position things.

But my issue here is with the state `is-closed`, the `.CardList-card` isn't closed, it's the `.Discussion` that's closed. To go with your approach you should use a modifier instead, `.CardList-card--translucent` seems good.

Another way is to move this css to the `.Discussion` and use it with `.Discussion.Discussion--closed`, I think the modifier is better for this also.

I'm with both options.